xilem_web/src/message.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright 2024 the Xilem Authors
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{any::Any, fmt::Debug, ops::Deref};
+
+/// A dynamically typed message for the [`View`] trait.
+///
+/// Mostly equivalent to `Box<dyn Any>`, but with support for debug printing.
+// We can't use intra-doc links here because of
```

Comment:
I guess I should fix this sentence fragment in both places...

Reply:
Yeah, I'm not really sure what you mean with it^^ (I suspect it has something to do with `dyn`...)

Reply:
Yeah; it's the `dyn Message`. Rustdoc doesn't seem to have a syntax to create an intra-doc-link to this item